core/blockchain.go

```diff
@@ -1655,7 +1655,7 @@ func (bc *BlockChain) InsertChain(chain types.Blocks) (res *ChainInsertResult) {
 			return
 		}
 		// Write state changes to database
-		_, err = bc.stateCache.CommitTo(bc.chainDb, false)
+		_, err = bc.stateCache.CommitTo(bc.chainDb, bc.config.IsAtlantis(block.Number()))
```

Comment:
why did you include the state trie clearing commit in this PR? https://github.com/etclabscore/go-ethereum/pull/6/commits/338747b6c726711e54d57aac9c5cce9c3865df82